### PR TITLE
Scripts/RuinsOfAhnQiraj: Crystal handling

### LINKED ISSUE #17315: [3.3.5] Boss: AQ20 Ossirian's crystals are not clickable.
**Description**:
Ossirian has a 300% damage buff which you disable when you click on a crystal.
The crystals spawn but are not clickable as a player. I looked at the code and their ID is 180619 which in the DB is set as faction 35, which makes it clickable by all players. When spawned manually using .gobject add, the crystal is clickable. When the boss spawns it, it is not.
**Current behaviour**: Tell us what happens
The crystals spawn but are not clickable as a player.
The crystals are clickable as a GM and do what they are intended to do when clicked.

**Expected behaviour**: Tell us what should happen instead
They should be clickable to a player.

**Steps to reproduce the problem**:
Engage Ossirian and see for yourself.

**Branch(es)**: 
3.3.5
71081c2576571a6e6d5a16179e6729843094a882

### PR DESCRIPTION
**Pull request by Faq**
https://wowwiki.fandom.com/wiki/Ossirian_the_Unscarred
https://tcubuntu.northeurope.cloudapp.azure.com/aowow/?npc=15339

**Changes proposed:**
+ make crystal usable with animation and despawn after use;
+ small code update;
+ spell rearrange;
+ define some magic numbers;
+ update crystal coordinates from sniffed data

**Target branch(es):** 3.3.5/master

- [x] 3.3.5
- [ ] master

**Issues addressed:** Closes #  (insert issue tracker number)
Closes #17315 

**Tests performed:** (Does it build, tested in-game, etc.)
Compiled and tested by using crystals

todo:
[+] update crystal spawn coords from #24681

### CODE REVIEW

**Comment by Faq** [comment]
ref #24681

File: src/server/scripts/Kalimdor/RuinsOfAhnQiraj/boss_ossirian.cpp [hunk 1]
```diff
@@ -63,28 +69,28 @@ enum Events
...
-    { -9406.73f, 1863.13f, 85.5558f, 0.0f }
+    { -9188.4404296875f, 1940.2099609375f, 85.6390991210937f, 3.17650008201599f },
+    { -9244.41015625f, 1808.97998046875f, 85.6390991210937f, 5.63741016387939f },
+    { -9248.41015625f, 1974.82995605469f, 85.6390991210937f, 5.89920997619629f },
+    { -9282.080078125f, 1887.33996582031f, 85.6390991210937f, 2.00712990760803f },
+    { -9299.73046875f, 1748.44995117187f, 85.6390991210937f, 1.44861996173859f },
+    { -9357.8603515625f, 1929.07995605469f, 85.6390991210937f, 1.06465005874634f },
+    { -9367.169921875f, 1780.89001464844f, 85.6390991210937f, 1.90241003036499f },
+    { -9383.2900390625f, 2012.68005371094f, 85.6511001586914f, 2.93214988708496f },
+    { -9406.099609375f, 1862.38000488281f, 85.6390991210937f, 6.2308201789856f },,
```
Comment (Shauren) [inline src/server/scripts/Kalimdor/RuinsOfAhnQiraj/boss_ossirian.cpp:84]: doesnt compile, you have two commas at the end here

Comment (Faq) [reply src/server/scripts/Kalimdor/RuinsOfAhnQiraj/boss_ossirian.cpp]: upsy, corrected.

**Comment by jackpoz** [comment]
Thanks for the PR :)

**Comment by illfated** [comment]
Good job, nice work. :)

**Pull request merged**